src/adapter/src/catalog/builtin.rs

```diff
@@ -3057,6 +3057,7 @@ pub const MZ_DATAFLOW_ARRANGEMENT_SIZES: BuiltinView = BuiltinView {
     AS
         SELECT
             mdod.dataflow_id AS id,
+            mo.id AS object_id,
```

Comment:
I'm pretty sure you'll need to add this to the group by clause (or perform some type of aggregation on it, which probably doesn't make much sense).